src/platforms/java/common/configuration/integrations/graphql.mdx

```diff
@@ -10,109 +10,111 @@ notSupported:
+Create a new instance of `SentryInstrumentation` and configure `instrumentation` when building a `GraphQL` instance.
+You may want to filter some of the errors by using `beforeSend` or an `EventProcessor` (see <PlatformLink to="/configuration/filtering/">Filters</PlatformLink>).
 
 ```java
 import graphql.GraphQL;
 import graphql.execution.SimpleDataFetcherExceptionHandler;
-import io.sentry.graphql.SentryDataFetcherExceptionHandler;
+import io.sentry.graphql.SentryGenericDataFetcherExceptionHandler;
+import io.sentry.graphql.SentryInstrumentation;
 
 SimpleDataFetcherExceptionHandler defaultExceptionHandler = new SimpleDataFetcherExceptionHandler();
-SentryDataFetcherExceptionHandler sentryExceptionHandler = new SentryDataFetcherExceptionHandler(defaultExceptionHandler);
+SentryGenericDataFetcherExceptionHandler sentryExceptionHandler = new SentryGenericDataFetcherExceptionHandler(defaultExceptionHandler);
 
 GraphQL graphQL = GraphQL.newGraphQL(...)
     // ...
     .defaultDataFetcherExceptionHandler(sentryExceptionHandler)
+    .instrumentation(new SentryInstrumentation(
+      // If you're not using our Spring integration, please provide NoOpSubscriptionHandler.getInstance() instead.
+      new SentrySpringSubscriptionHandler(),
+      // Set this to false when using Spring WebMVC
+      true
+    ))
     .build();
 ```
 
 ```kotlin
 import graphql.GraphQL
 import graphql.execution.SimpleDataFetcherExceptionHandler
-import io.sentry.graphql.SentryDataFetcherExceptionHandler
+import io.sentry.graphql.SentryGenericDataFetcherExceptionHandler
+import io.sentry.graphql.SentryInstrumentation
 
 val defaultExceptionHandler = SimpleDataFetcherExceptionHandler()
-val sentryExceptionHandler = SentryDataFetcherExceptionHandler(defaultExceptionHandler)
+val sentryExceptionHandler = SentryGenericDataFetcherExceptionHandler(defaultExceptionHandler)
 
 val graphql = GraphQL.newGraphQL()
   .defaultDataFetcherExceptionHandler(sentryExceptionHandler)
+	.instrumentation(SentryInstrumentation(
+    // If you're not using our Spring integration, please provide NoOpSubscriptionHandler.getInstance() instead.
+    SentrySpringSubscriptionHandler(),
+    // Set this to false when using Spring WebMVC
+    true
+  ))
   .build()
 ```
 
-## Capture Performance
-
 <Note>
 
-Capturing transactions requires that you first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink> if you haven't already.
+If you've been using the now deprecated `SentryDataFetcherExceptionHandler`, please upgrade to `SentryGenericDataFetcherExceptionHandler` and also make sure `SentryInstrumentation` is configured to have more exceptions captured, more detailed exceptions, breacrumbs and better hub propagation. You may want to filter some of the errors by using `beforeSend` or an `EventProcessor` (see <PlatformLink to="/configuration/filtering/">Filters</PlatformLink>).
```

Comment:
```suggestion
The `SentryDataFetcherExceptionHandler` has been deprecated. Please upgrade to `SentryGenericDataFetcherExceptionHandler` and make sure `SentryInstrumentation` is configured to have more exceptions captured, more detailed exceptions, breadcrumbs, and better hub propagation. You may want to filter  the errors by using `beforeSend` or an `EventProcessor` (read more about <PlatformLink to="/configuration/filtering/">Filters</PlatformLink>).
```

This seems like a daunting list of thing to "make sure of..." do you think it's clear to users why they need to do this and how to do it? If not, maybe we can write a separate section about it?

Reply:
It's just meant as a side note for people who already used our GraphQL integration before the latest changes. This should put them on the right track to upgrade plus some reasons why in case they missed the changelog entry.